### PR TITLE
Update to the way we consume SUPPORTED_LOCALES

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -48,6 +37,7 @@ function getListLocales() {
         reject(e);
       });
     } else {
+      supportedLocales = supportedLocales.split(",").map(item => item.trim());
```
Comment (Pomax) [inline index.js]: you can probably do `split(/\s*,\s*/g)` so you don't need the trim

Comment (ScottDowne) [inline index.js]: Heh, yeah, I was trying to avoid regex on purpose.

Comment (Pomax) [inline index.js]: in that case, why not just keep it documented that the list should be CSV, no spaces, and not bother to trim? Saves a bit of work.

Comment (ScottDowne) [inline index.js]: I'll do the regex before dumping this on documentation :)

Comment (alanmoo) [inline index.js]: I'd say this is a reasonable amount of defensive programming to include and be worth a quick loop.

**Pull request merged**